README.rst

```diff
@@ -59,12 +59,50 @@ Next, we prepare for object pose fitting, by running ElasticFusion and formattin
 
 	run_prep
 
-Next, launch the object alignment tool.  See Pipeline_Instructions_ for how to use the alignment tool:
+Next, launch the object alignment tool and follow the three steps:
 
 ::
 
 	run_alignment_tool
 
+1. 	Align the reconstructed point cloud:
+
+	- Open measurement panel (View -> Measurement Panel), then check Enabled in measurement panel
+	- Use (shift + click) and click two points: first on the surface of the table, then on a point above the table
+	- Open Director terminal with F8 and run::
+
+		gr.rotateReconstructionToStandardOrientation()
+
+	- Close the ``run_alignment_tool`` application (ctrl + c) and rerun.
+
+2. 	Segment the pointcloud above the table
+
+	- Same as above, use (shift + click) and click two points: first on the
+	surface of the table, then on a point above the table
+	- Open Director terminal with F8 and run::
+
+		gr.segmentTable()
+		gr.saveAboveTablePolyData()
+
+	- Close the ``run_alignment_tool`` application (ctrl + c) and rerun.
+
+3. 	Align each object and crop point clouds.
+
+	- Assign the current object you're aligning, e.g.::
+	
+		objectName = "drill"
+
+	- Launch point cloud alignment::
+
+	    gr.launchObjectAlignment(objectName)
+
+	  This launches a new window. Click the same three points in model and on pointcloud. Using :code:`shift + click` to do this. After you do this the affordance should appear in main window using the transform that was just computed. You can crop the pointcloud using the alignments we just got by calling::
+
+		gr.cropPointCloudUsingAlignedObject(objectName=objectName)
```

Comment:
I don't think that this command is necessary. After you complete the object alignment popup window the object should be aligned.

Reply:
Done. Removed `gr.cropPointCloudUsingAlignedObject`. Thanks!
(Also, added a small comment about re-doing fitting.)